iqmath/test_apps/README.md

```diff
@@ -0,0 +1,2 @@
+| Supported Targets | ESP32 | ESP32-C2 | ESP32-C3 | ESP32-C6 | ESP32-H2 | ESP32-P4 | ESP32-S2 | ESP32-S3 |
```

Comment:
is this table added by pre-commit or manually? iqmath should work on all targets, so this table doesn't make too much sense.